pheps/phep-0001.md

```diff
@@ -0,0 +1,338 @@
+
+# Transferring PHEP Ownership
+<a name="transferring-phep-ownership"></a>
+It occasionally becomes necessary to transfer ownership of PHEPs to a new author. It is preferable to retain the original author as a co-author of the transferred PHEP, at their discretion. A good reason to transfer ownership is because the original author no longer has the time or interest in updating it or following through with the PHEP process, or is unreachable. A bad reason to transfer ownership is because the author doesn't agree with the direction of the PHEP. One aim of the PHEP process is to try to build consensus around a PHEP, but if that's not possible, an author can always submit a competing PHEP.
+
+If you are interested in assuming ownership of a PHEP, mention the original author and `@heliophysicsPy/phep-editors` in a comment on the PHEP's pull request. If the original author doesn't respond in a timely manner, the PHEP editors will make a unilateral decision, which can be reversed.
+
+# PHEP Editor Responsibilities & Workflow
+<a name="phep-editor-responsibilities-workflow"></a>
+A PHEP editor must be added to the `@heliophysicsPy/phep-editors` group on GitHub and must watch the `standards` repository.
+
+For each new PHEP that comes in, an editor does the following:
+
+* Read the PHEP to check if it is ready: sound and complete. The ideas must make technical sense, even if they don't seem likely to be accepted.
+
+* The title should accurately describe the content.
+
+* The PHEP's type is correctly labeled ("Standards Track", "Informational", or "Process"), and its status is "Draft".
+
+* The file name extension is correct (i.e. `.md`).
+
+* Skim the PHEP for obvious defects in language (spelling, grammar, sentence structure, etc.) or formatting. Editors may correct problems themselves, but are not required to do so.
+
+* If a project is portrayed as benefiting from or supporting the PHEP, make sure there is some direct indication from the project included to make the support clear.
+
+If the PHEP isn't ready, an editor will send it back to the author for revision, with specific instructions.
+
+Once the PHEP is ready for discussion, a PHEP editor will:
+
+* Assign a PHEP number (almost always the next available number).
+
+* Ask the author to update the PHEP with the PR link, mark the PR ready for review, and post an announcement.
+
+PHEP editors do not make judgments on the content or merits of PHEPs. They merely do the administrative & editorial part. They may participate in the community discussion of a PHEP but have no obligation to be advocates.
+
+Disagreements with editors must be mediated by PyHC leadership (as documented in [PHEP audience](#phep-audience)) and they may impose sanctions, including removal of the editor role.
+
+# PHEP 1 Acceptance
+<a name="phep-1-acceptance"></a>
+This PHEP is subject to the same review and acceptance process that it proposes.
```

Comment:
@wafels raised the question of testing this process and PHEP-1 is basically its own test.